backend/src/src-cvesearch/src/main/java/org/eclipse/sw360/cvesearch/datasource/heuristics/SearchLevels.java

```diff
@@ -168,7 +168,7 @@ public SearchLevels addGuessingSearchLevels(CveSearchApi cveSearchApi, int vendo
 
         return vendorProductList.stream()
                 .map(match -> new NeedleWithMeta(cpeBuilder.apply(match.getNeedle()),
-                        "heuristic (dist. " + match.getDistance() + ")"))
+                        "heuristic (dist. " + (useVersionInformation ? "0" : "1") + match.getDistance() + ")"))
```

Comment:
Yes, i want to make it a two-digit number starting with a `1` or a `0`, where `0` means that the version has matched.